posthog/settings.py

```diff
@@ -450,7 +450,7 @@ def get_list(text: str) -> List[str]:
 # https://docs.djangoproject.com/en/2.2/ref/settings/#databases
 
 if TEST or DEBUG:
-    DATABASE_URL = os.getenv("DATABASE_URL", "postgres://localhost:5432/posthog")
+    DATABASE_URL = os.getenv("DATABASE_URL", "postgres://posthog:posthog@localhost:5432/posthog")
```

Comment:
Not sure about this change, by default some local installations have an empty password and the default OS username (e.g. [Postgres.app](https://postgresapp.com/documentation/configuration-general.html))

Reply:
This got updated in master anyway...